components/sandbox/sandbox-platform.tf

```diff
@@ -1,16 +1,21 @@
-data "local_file" "configuration" {
-  filename = "${path.cwd}/../../environments/sandbox.yml"
+data "local_file" "sandbox-config" {
```

Comment:
doesn't follow naming conventions
https://www.terraform-best-practices.com/naming#general-conventions

`Use _ (underscore) instead of - (dash) everywhere (in resource names, data source names, variable names, outputs, etc).`